hack/lib/etcd.sh

```diff
@@ -112,8 +112,8 @@ kube::etcd::install() {
 
     cd "${KUBE_ROOT}/third_party" || return 1
     if [[ $(readlink etcd) == etcd-v${ETCD_VERSION}-${os}-* ]]; then
-      kube::log::info "etcd v${ETCD_VERSION} already installed at path:"
-      kube::log::info "$(pwd)/$(readlink etcd)"
+      kube::log::info "etcd v${ETCD_VERSION} already installed. To use:"
+      kube::log::info "export PATH=$(pwd)/etcd:\${PATH}"
```

Comment:
`${PATH}` should be escaped here if it's a possible instruction. Also, this is not likely correct.. you need to readlink to discover the right path, right?

Reply:
@cblecker 
Thanks very much for your quick review and comments.

For your first comments "**${PATH} should be escaped here if it's a possible instruction**", can you help give an example if possible? "\\" is used as "Escape character" before "${PATH}" in my change, do I miss something?

For your second comments, seems we needn't update it, we can just use "**etcd**" instead of "**readlink etcd**" directly because:
1. The result of "readlink etcd" is like "**etcd-v3.3.10-linux-amd64**", and the script already links it to the "**etcd**" when installing  the etcd, they are for the same folder.

    ll /opt/kubernetes/third_party/
    total 40
     ......
    lrwxrwxrwx  1 root    root    24 Apr 15 10:05 **etcd -> etcd-v3.3.10-linux-amd64**/
    drwxr-xr-x  3 6810230 users 4096 Oct 10  2018 etcd-v3.3.10-linux-amd64/

2. This is to keep the similar output for the cases that "etcd is already installed" and "etcd is just installed", the following codes are copied from the end of "etcd.sh", it will ask user to "export PATH=.../**etcd**:${PATH}" instead of "export PATH=.../**etcd-v3.3.10-linux-amd64**:${PATH}" for the case "etcd is just installed", and this is why I use "etcd" directly in the codes.

    `kube::log::info "etcd v${ETCD_VERSION} installed. To use:"`
    `kube::log::info "export PATH=$(pwd)/etcd:\${PATH}"`

Reply:
`"export PATH=\"$(pwd)/etcd:\${PATH}\""` should print the right escaped syntax:
```
$ echo "export PATH=\"$(pwd)/etcd:\${PATH}\""
export PATH="/opt/k8sgopath/src/k8s.io/kubernetes/third_party/etcd:${PATH}"
```

You're right though, that using just `/etcd` here is fine, because it's a symlink. Can you please update the syntax on L136 to match this too?

Thanks!

Reply:
@cblecker
Thanks, and I will update the codes with your suggestion. 
I'm not available to change the codes now, I will handle it several hours later.

Reply:
@cblecker, codes updated, please help review again.

The output for the first install:
# ./install-etcd.sh
... ...
etcd v3.3.10 installed. To use:
export PATH="/opt/kubernetes/third_party/etcd:${PATH}"

The output for the re-install:
# ./install-etcd.sh
etcd v3.3.10 already installed. To use:
export PATH="/opt/kubernetes/third_party/etcd:${PATH}"